resource_manager.py

```diff
@@ -30,10 +30,17 @@ def _get_checkpoint_model_name(self, model_name, epoch):
     def _get_checkpoint_file_template(self, model_name):
         return model_name + "_epoch_{epoch}.h5"
 
-    def get_epoch_save_callback(self, model_name, period):
+    def get_epoch_save_period(self):
+        return [0, 1, 2, 3, 8, 40, 90]
+
```

Comment:
Note that this used to be `[0,1,2,3,8,40,90,100]` but there's no need for epoch `100` because that's the epoch saved under `end`. The `on_epoch_end` callback isn't invoked for epoch `100` so this caused the `ResourceManager` to complain it couldn't load `mnist_epoch_100.h5`